src/zmq/zmqnotificationinterface.cpp

```diff
@@ -160,3 +162,19 @@ void CZMQNotificationInterface::SyncTransaction(const CTransaction& tx, const CB
         }
     }
 }
+
+void CZMQNotificationInterface::BlockConnected(const std::shared_ptr<const CBlock>& pblock, const CBlockIndex* pindexConnected, const std::vector<CTransactionRef>& vtxConflicted)
```

Comment:
To help others review, these two functions seem to mesh with current behavior. I don't see how they could possibly be usable by any client, but there ya go.